### PR TITLE
Update readme style

### PR DESCRIPTION
**Pull request by JoshOrndorff**
Goals:
* be concise
* follow https://github.com/subst…rate-developer-hub/substrate-developer-hub.github.io/blob/source/CONTRIBUTING.md#documentation-style

@jimmychu0807 There are a few details I didn't understand yet, and I've put TODO items there. Could you please fill in those details? Also feel free to revert changes where you prefer your original style.

### CODE REVIEW

**Comment by shawntabrizi** [comment]
We should add a note in the `Configuration` setting that these are "configurations for production environments".

For example, the configuration tells the user to turn off the "test accounts", but they probably want that for any dev stuff they are doing.

File: README.md [hunk 1]
```diff
@@ -1,25 +1,50 @@
...
+## Usage
+You can start the template in development mode to connect to a locally running node
+```bash
+yarn start
+```
+
+You can also start it in production mode...
+TODO @jimmychu What are the details about this again?
 
 ## Configuration
```
Comment (shawntabrizi) [inline README.md]:
```suggestion
## Production Configuration
```

File: README.md [hunk 2]
```diff
@@ -1,25 +1,50 @@
...
+precedence than `common.json`.
+* `production.json` affects only the production environment with higher
+precedence than `common.json`.
+TODO @jimmychu what's `test.json`? Can you please write about it or remove it?
+
+More on
+[React environment variables](https://create-react-app.dev/docs/adding-custom-environment-variables).
 
-When writing and deploying an front end with your own node, you likely want to change:
+When writing and deploying your own front end, you will likely change:
```
Comment (shawntabrizi) [inline README.md]:
```suggestion
When writing and deploying your own front end, you should configure:
```

File: README.md [hunk 3]
```diff
@@ -1,25 +1,50 @@
...
+```
+
+## Usage
+You can start the template in development mode to connect to a locally running node
+```bash
+yarn start
+```
+
+You can also start it in production mode...
+TODO @jimmychu What are the details about this again?
```
Comment (shawntabrizi) [inline README.md]: `yarn build` triggers the production settings to kick in.

Comment (JoshOrndorff) [reply README.md]: Added those details. I'm open to suggestions on how to format that part.

File: README.md [hunk 4]
```diff
@@ -1,25 +1,53 @@
...
-loaded in both development and production environments, and then the environment
-specific JSON file (with higher priority). More on React enviornment variables
-can be seen [here](https://create-react-app.dev/docs/adding-custom-environment-variables).
+The template's configuration is in three files in the `src/config` directory.
+* `common.json` affects both development and production environments.
+* `development.json` affects only the development environment with higher
+precedence than `common.json`.
+* `production.json` affects only the production environment with higher
+precedence than `common.json`.
+TODO @jimmychu what's `test.json`? Can you please write about it or remove it?
```
Comment (jimmychu0807) [inline README.md]: `test.json` is used when running `yarn test` in the CI workflow.

Comment (JoshOrndorff) [reply README.md]: Thanks for adding the info :)

**Pull request merged**